R/splitOn.R

```diff
@@ -13,26 +13,33 @@
 #'   Split by cols is not encouraged, since this is not compatible with 
 #'   storing the results in \code{altExps}.
 #'
-#' @param keep_reducedDims \code{TRUE} or \code{FALSE}: Should the
+#' @param keep.reduceddims \code{TRUE} or \code{FALSE}: Should the
 #'   \code{reducedDims(x)} be transferred to the result? Please note, that this
 #'   breaks the link between the data used to calculate the reduced dims.
-#'   (By default: \code{keep_reducedDims = FALSE})
+#'   (By default: \code{keep.reduceddims = FALSE})
 #'   
-#' @param update_rowTree \code{TRUE} or \code{FALSE}: Should the rowTree be updated
+#' @param keep_reducedDims Deprecated. Use \code{keep.reduceddims} instead.
+#'   
+#' @param update.rowtree \code{TRUE} or \code{FALSE}: Should the rowTree be updated
```

Comment:
This could be agglomerate.tree to harmonize between functions. agglomerate* functions use agglomerate.tree name. Here the parameter functions similarly